uvicorn/server.py

```diff
@@ -217,7 +217,7 @@ async def main_loop(self) -> None:
         should_exit = await self.on_tick(counter)
         while not should_exit:
             counter += 1
-            counter = counter % 864000
+            counter %= 864000
```

Comment:
Can we revert this one? 🙏 
```suggestion
            counter = counter % 864000
```